### ISSUE TITLE
NoSpotifyPause not working

### ISSUE THREAD

**Issue by ienooo** [bepvte/bd-addons#36]
**Plugin name and version**
NoSpotifyPause v0.0.3

**Describe the bug**
The Plugin simply doesn't seem to work for me. Whenever I speak for too long in a vc, my spotify pauses and this pause notification on discord pops up.

**Environment**

- Are you using Discord Canary or PTB? 
- > No, using BD Core v1.8.2 and Discord v1.0.9007
- Are you using litecord or some other unsupported and weird client mod? If so, the issue will be closed and ignored. 
- > No
- [x] I have tested to verify that this does not occur when running without betterdiscord.
      (To test without betterdiscord, run the betterdiscord installer and press uninstall. Your plugins and settings wont be deleted)
- [x] I have tested with no other plugins to confirm that the issue is with the specific plugin in question

**Comment by bepvte** [comment]
Hi, is ZeresPluginLibrary loaded and updated to version v2.0.9?

**Comment by ienooo** [comment]
sorry but yea, it was/is on version 2.0.9

**Comment by bepvte** [comment]
Is it still broken?

**Comment by ienooo** [comment]
just tried it out. still got the notification that the playback has been paused but spotify did not pause this time. maybe bdc just didnt load the plugin properly or something else.

**Comment by ienooo** [comment]
Just letting you know. Happened again (v0.0.3). 

The plugin is toggled on but my spotify got paused twice in a row. The plugin seems to be only working sometimes but I believe if it isn't working it wont work for the entire Discord session. After restarting Discord the problem has been fixed but this has happened multiple times now. 

Restarting Discord is a pretty simple fix but I figured I'd let u know anyways
